### ISSUE TITLE
Install vue-axios with no module bundler support (e.g. webpack)

### ISSUE THREAD

**Issue by acsor** [imcvampire/vue-axios#54]
The project README suggests to install this dependency with NPM

```
$ npm install --save axios vue-axios
```

but for the project I'm working on, I'm not allowed to rely on webpack. Under these conditions, can I follow the same procedure as listed in the README for installing `vue-axios` or is there something else I can do? Further, what do you mean when you say the following - what is the `document` you refer to?

> Just add 3 scripts in order: vue, axios and vue-axios to your document.

**Comment by xiaofan9** [comment]
You can try to use the script tag src to introduce, https://unpkg.com/vue-axios@2.1.5/dist/vue-axios.min.js

**Comment by acsor** [comment]
Yeah, that's what I did just after posting this question. It's probably a question no longer (strictly) related `vue-axios`, but how could I procede if I wanted to serve the library from my Node server instead?

**Comment by xiaofan9** [comment]
for node, I recommend using axios instead of vue-axios

**Comment by acsor** [comment]
TL;DR [unpkg](https://unpkg.com/) was what I was looking for.
